litmus-portal/frontend/src/pages/MonitoringDashboardPage/styles.ts

```diff
@@ -66,50 +58,158 @@ const useStyles = makeStyles((theme) => ({
     },
     minWidth: '10rem',
     height: '2.5rem',
   },
 
   expDiv: {
     display: 'flex',
-    flexDirection: 'row',
   },
 
   btnText: {
     fontWeight: 500,
   },
 
-  icon: {
-    width: '6rem',
-    height: '6rem',
+  selectDate: {
+    display: 'flex',
+    height: '2.9rem',
+    minWidth: '9rem',
+    marginLeft: theme.spacing(3.75),
+    border: `0.1px solid ${theme.palette.border.main}`,
+    background: theme.palette.background.paper,
+    borderRadius: 4,
+    textTransform: 'none',
+  },
+
+  displayDate: {
+    width: '100%',
+    color: theme.palette.text.primary,
+  },
+
+  rangeSelectorIcon: {
+    width: '0.625rem',
+    height: '0.625rem',
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(-1),
   },
 
-  modalHeading: {
-    marginTop: theme.spacing(3.5),
-    fontSize: '2.25rem',
-    marginBottom: theme.spacing(4.5),
+  rangeSelectorClockIcon: {
+    width: '0.825rem',
+    height: '0.825rem',
+    marginLeft: theme.spacing(-1),
+    marginRight: theme.spacing(1),
   },
 
-  modalBody: {
-    marginBottom: theme.spacing(4.5),
+  rangeSelectorPopover: {
+    marginTop: theme.spacing(37.5),
   },
 
-  flexButtons: {
+  formControl: {
+    width: '9rem',
+    marginLeft: theme.spacing(1.5),
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(3.75),
+    },
+    '& .MuiSelect-outlined': {
+      padding: '0.925rem',
+      '&:focus': {
+        borderColor: theme.palette.primary.main,
+      },
+      '& .MuiInputLabel-root': {
+        color: `${theme.palette.text.hint} !important`,
+        marginTop: `${theme.spacing(2)} !important`,
+      },
+    },
+  },
+
+  inputLabel: {
+    color: theme.palette.text.hint,
+    marginTop: theme.spacing(-1),
+    '&.MuiInputLabel-shrink': {
+      marginTop: theme.spacing(1),
+    },
+  },
+
+  menuListItem: {
+    color: theme.palette.text.hint,
+    height: '2.4rem',
+  },
+
+  menuListItemSelected: {
+    height: '2.4rem',
+    background: `${theme.palette.primary.light} !important`,
+    color: theme.palette.secondary.contrastText,
+    '&:hover': {
+      background: `${theme.palette.primary.light} !important`,
+    },
+  },
+
+  refreshIcon: {
+    marginRight: theme.spacing(1),
+  },
+
+  refreshText: {
+    marginTop: theme.spacing(-2.85),
+    marginLeft: theme.spacing(3.5),
+  },
+
+  headerInfoText: {
+    fontWeight: 500,
+    fontSize: '1rem',
+    lineHeight: '150%',
+    letterSpacing: '0.02em',
+    width: '35%',
+    marginLeft: theme.spacing(2.5),
+  },
+
+  controls: {
+    marginRight: theme.spacing(2.25),
     display: 'flex',
-    flexDirection: 'row',
-    justifyContent: 'space-evenly',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      marginRight: 0,
+    },
   },
 
-  loader: {
-    padding: '22%',
+  chaosTableSection: {
+    padding: theme.spacing(2.5, 2, 0),
   },
+}));
 
-  modal: {
-    padding: theme.spacing(15, 0),
+export const useOutlinedInputStyles = makeStyles((theme: Theme) => ({
```

Comment:
can you provide screenshots for this input component please.

Reply:
sure, we are to override these from MUI select's label, as we are doing at all other places.
<img width="175" alt="Screenshot 2021-04-06 at 1 04 46 PM" src="https://user-images.githubusercontent.com/50652677/113692860-9f5bb300-96eb-11eb-9cb1-54797af39af4.png">